contrib/setup-z3.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+# issues creating a static binary with z3
+# (related to pthread)
+# see https://github.com/Z3Prover/z3/issues/4554
```

Comment:
I am not sure what does this comment supposed to mean for the reader.

Reply:
Ahh yeah was meant for me, not for other people but that isn't very useful. Fixed with e72a6cb5875bb1f9c4248417ec37bafacaee449e